styles/style.css

```diff
@@ -133,7 +143,38 @@ p {
   font-size: 1.2rem;
   margin-bottom: 2rem;
 }
+.red-button{
+  color: white;
+  background-color: orangered;
+  border-radius: 5px;
+  border: 0px;
+  padding: 10px;
+}
+.red-button:hover{
+  color:orangered;
+  background-color: white;
+  border-radius: 5px;
+  border: 0px;
+  padding: 10px;
+}
 
+.white-button{
```

Comment:
Consider what can be simplified in you button styles - it feels like a lot of repetition. Try to remove code duplication if possible